docs/user/azure/install_upi_azurestack.md

```diff
@@ -276,7 +276,7 @@ storage blob.
 First, download and decompress the VHD, note that the decompressed file is 16GB:
 
 ```sh
-$ export COMPRESSED_VHD_URL=$(curl -s https://raw.githubusercontent.com/openshift/installer/release-4.9/data/data/rhcos-amd64.json | jq -r '(.baseURI + .images.azurestack.path)')
+$ export COMPRESSED_VHD_URL=$(openshift-install coreos print-stream-json | jq -r '.architectures.x86_64."rhel-coreos-extensions"."azure-disk".url')
```

Comment:
Azure Stack uses a different image than Azure (and this is using the Azure disk). I think the jq command should be:

```
openshift-install coreos print-stream-json | jq -r '.architectures.x86_64.artifacts.azurestack.formats."vhd.gz".disk.location'
```